File: benchmarks/pb_smallfunc/mono.php
<?php

function multiply_and_add($a, $b, $c) {
	return $a + $b * $c;
}

function outer($outer, $inner) {
	$iter_countdown = $outer * 10;
	$i = $iter_countdown;
	while($i > 0) {
		$i = multiply_and_add($i, 1, -1);
	}
}

function run_iter($n) {
	outer($n, 10000);
}

run_iter(10000000);

?>

File: benchmarks/pb_smallfunc/mono.py

def multiply_and_add(a, b, c):
    return a + b * c

def outer(outer, inner_unused):
    iter_countdown = outer * 10
    i = iter_countdown
    while i > 0:
        i = multiply_and_add(i, 1, -1)

def run_iter(n):
    outer(n, 10000)
